library/std/src/io/error.rs

```diff
@@ -156,6 +191,62 @@ pub enum ErrorKind {
     /// [`Ok(0)`]: Ok
     #[stable(feature = "rust1", since = "1.0.0")]
     WriteZero,
+    /// The underlying storage (typically, a filesystem) is full.
+    ///
+    /// This does not include out of quota errors.
+    #[stable(feature = "io_error_unix_more", since = "1.51.0")]
+    NoStorageSpace,
```

Comment:
This looks like window's [`ERROR_OUTOFMEMORY`](https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499-)

Reply:
Are you sure?  Isn't that out of *memory* rather than disk space?  On same page I found `ERROR_DISK_FULL` and `ERROR_HANDLE_DISK_FULL`.  For now I will map both of those to `NoStorageSpace`